test/Tests/src/Semantic/Error_Spec.enso

```diff
@@ -446,6 +446,17 @@ spec =
             e = Error.throw (My_Error_Wrapper.Error Nothing) . catch Nothing
             e . should_equal Nothing
 
+        Test.specify "Can unwrap a wrapped error" <|
```

Comment:
Missing case of unwrapping an unwrapped error (aka `identity`)?

Reply:
Added.